### PR TITLE
feat(accessibility): add new labels for input buttons

### PR DESCRIPTION
**Pull request by ardasnturk**
- added `accessibilityLabel` for `searchIcon` and `clearIcon` used in search input for users using `VoiceOver` feature
- added `accessibilityLabel` for the show or hide password button used in the password input for users using the `VoiceOver` feature
- added `accessibilityLabels` section to texts for use in new accessibility features

SVA-899

### CODE REVIEW

File: src/components/BB-BUS/TextSearch.js [hunk 1]
```diff
@@ -3,10 +3,12 @@ import React, { memo } from 'react';
 import { StyleSheet, View } from 'react-native';
 import { SearchBar } from 'react-native-elements';
 
-import { colors, normalize } from '../../config';
+import { colors, consts, normalize, texts } from '../../config';
 import { Label } from '../Label';
 import { WrapperHorizontal } from '../Wrapper';
 
+const a11yText = consts.a11yLabel;
```
Comment (donni106) [inline src/components/BB-BUS/TextSearch.js:10]: why not `const { a11yLabel } = consts;`?

Comment (ardasnturk) [reply src/components/BB-BUS/TextSearch.js:10]: It is used in two different ways in the code. We will need to refactor the whole code.

Comment (ardasnturk) [reply src/components/BB-BUS/TextSearch.js]: https://github.com/smart-village-solutions/smart-village-app-app/commit/59c9c808be9e127b5416245dcda7109ca46cf86c

Comment (donni106) [reply src/components/BB-BUS/TextSearch.js]: correct, but than let's use the favorite one if we add new code

File: src/components/form/InputSecureTextIcon.tsx [hunk 2]
```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import { TouchableOpacity } from 'react-native';
 
-import { colors, Icon } from '../../config';
+import { colors, consts, Icon, texts } from '../../config';
+
+const a11yText = consts.a11yLabel;
```
Comment (donni106) [inline src/components/form/InputSecureTextIcon.tsx:6]: you missed it here

Comment (ardasnturk) [reply src/components/form/InputSecureTextIcon.tsx]: https://github.com/smart-village-solutions/smart-village-app-app/commit/673dec6e31683527ba307138ccec0ebce57e64fd

**Review by donni106 — APPROVED** [review]

**Pull request merged**